api/v1beta1/policyautomation_types.go

```diff
@@ -14,18 +14,21 @@ type PolicyAutomationSpec struct {
 	// +kubebuilder:validation:Required
 	PolicyRef string `json:"policyRef"`
 	// Mode decides how automation is going to be triggered
-	// +kubebuilder:validation:Enum={once,disabled}
-	// +kubebuilder:validation:Required
-	Mode string `json:"mode"`
+	Mode PolicyAutomationMode `json:"mode"`
 	// EventHook decides when automation is going to be triggered
 	// +kubebuilder:validation:Enum={noncompliant}
 	// +kubebuilder:validation:Required
-	EventHook   string `json:"eventHook,omitempty"`
-	RescanAfter string `json:"rescanAfter,omitempty"`
+	EventHook            string `json:"eventHook,omitempty"`
+	RescanAfter          string `json:"rescanAfter,omitempty"`
+	DelayAfterRunSeconds uint   `json:"delayAfterRunSeconds,omitempty"`
```

Comment:
Could you please set the `// +kubebuilder:validation:Minimum` marker so that there is API validation that the value is at least 0 (i.e. no negative numbers)?